Sources/Sentry/SentryScope.m

```diff
@@ -496,7 +496,7 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
         event.fingerprint = fingerprints;
     }
 
-    if (event.breadcrumbs == nil) {
+    if (event.breadcrumbs == nil && ![event.type isEqualToString:SentryEnvelopeItemTypeReplayVideo]) {
```

Comment:
@romtsn, @bruno-garcia
Apart from breadcrumb, should we clean replay event of any other data that usually goes with events?

Reply:
I dunno if you set these ones right now, but I'm clearing them out:
```
replayEvent.extra = nil;
replayEvent.threads = nil;
replayEvent.debugMeta = nil;
```

Reply:
Perfect! I will just save some cycles and not even copy them to the event.